SPEC.md

```diff
@@ -1,12 +1,46 @@
 # Code Climate Engine Specification
 
-**Note: This specification is a living, versioned document. We welcome your participation and appreciate your patience as we finalize the platform.**
+## Abstract
+
+This specification describes the contract that must be followed by an engine to ensure compatability with the rest of the tools in the Code Climate static analysis ecosystem. It defines several aspects of Engine operation including input, outputs, packaging, as well as performance and security restrictions.
+
+## Status of this document
+
+The current version of this specification is [in the repository](https://github.com/codeclimate/spec/blob/master/VERSION).  This specification is versioned using [Semantic Versioning](http://semver.org/).
+
+Engines declare the version of the specification they are compatible with in the manifest file, described below.
```

Comment:
This sentence still refers to the `engine.yml` as  a "manifest file". Should be "engine specification file" now, I guess?